core/modules/layout/includes/block.text.inc

```diff
@@ -79,6 +79,10 @@ class BlockText extends Block {
         '#description' => t('If enabled, this block will be made reusable across different layouts and be listed in on the !block_link page.', array('!block_link' => l(t('Custom blocks'), 'admin/structure/block'))),
         '#weight' => 1,
       );
+      // Add a notice for translation.
+      if (module_exists('language')) {
+        $form['convert']['reusable']['#description'] .= '<br />' . t('To make this block translatable, it must be made reusable as well.');
```

Comment:
Let's use the same translation string as we use in the normal form (whatever that becomes).